.editorconfig

```
@@ -1,17 +1,350 @@
-# EditorConfig is awesome: https://EditorConfig.org
+### From: https://github.com/dotnet/aspnetcore/blob/main/.editorconfig
+### Last sync: 120b944
+### Disabled: IDE0073
+; EditorConfig to support per-solution formatting.
+; Use the EditorConfig VS add-in to make this work.
+; http://editorconfig.org/
+;
+; Here are some resources for what's supported for .NET/C#
+; https://kent-boogaart.com/blog/editorconfig-reference-for-c-developers
+; https://learn.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference
+;
+; Be **careful** editing this because some of the rules don't support adding a severity level
+; For instance if you change to `dotnet_sort_system_directives_first = true:warning` (adding `:warning`)
+; then the rule will be silently ignored.
 
-# top-most EditorConfig file
+; This is the default for the codeline.
 root = true
 
 [*]
-end_of_line = lf
-insert_final_newline = true
+indent_style = space
```

We have been using tabs in all .cs files why change it to space?

You changed some tabs to spaces in `UT4MasterServer.Services/Hosted/ApplicationStartupService.cs` but most files still use tabs.

I would prefer to stick with tabs.

```suggestion
indent_style = tabs
```